src/i18n/locale/en.json

```diff
@@ -325,13 +325,13 @@
       },
       "reader_type": {
         "label": {
-          "continuous_horizontal_ltr": "Horizontal (LTR)",
-          "continuous_horizontal_rtl": "Horizontal (RTL)",
+          "continuous_horizontal_ltr": "Horizontal (LTR) →",
+          "continuous_horizontal_rtl": "Horizontal (RTL) ←",
```

Comment:
nice touch, maybe we could use emoji?

Reply:
Certainly. It may also be possible to find something for "double page".

Reply:
Would end up on the wrong side RTL?

Reply:
you're right, didn't think about that